src/stacks-hierarchy/Root_ChooseTicketReceiverScreen/navigation-types.ts

```diff
@@ -0,0 +1,5 @@
+import {Root_PurchaseConfirmationScreenParams} from '../Root_PurchaseConfirmationScreen';
+
+export type Root_ChooseTicketReceiverScreenParams = {
+  rootPurchaseConfirmationScreenParams: Root_PurchaseConfirmationScreenParams;
+};
```

Comment:
I think we shouldn't do it like this. I think it is ok to create typing for params based on other screen params, but I think it is too tight coupling to have actual field names based on other screens.

 Maybe rather just do:
```ts
Root_ChooseTicketReceiverScreenParams = Root_PurchaseConfirmationScreenParams;
``` 

Or maybe:
```ts
Root_ChooseTicketReceiverScreenParams = Omit<Root_PurchaseConfirmationScreenParams, 'phoneNumber'>;
``` 

is more correct?

Reply:
I was also thinking the same regarding coupling, was looking for a less coupling solution but can't find any when I was working on this part.

I'll try to use the ones that you suggested and see if it works. Thanks for the suggestion!

Reply:
I guess the suggestion with `Omit` is the best one

Reply:
done!